modules/ROOT/pages/configure-acm.adoc

```diff
@@ -480,8 +475,7 @@ If you don't configure an approval process, self-registered users will become me
 . Select *Use Standard Setup Wizard* in the *Create New Approval Process* drop-down.
 . In the wizard, enter the following information:
 
-.. In *Step 1. Enter Name and Description*, enter the following information:
-... *Process Name*: `Approve Registration`.
+.. In *Step 1. Enter Name and Description*, enter the following information: *Process Name*: `Approve Registration`.
```

Comment:
```suggestion
.. In *Step 1. Enter Name and Description*, enter the following information: 

   *Process Name*: `Approve Registration`
```

GLOBAL reformatting suggestion; I will stop marking for these double-colon sentences.

Reply:
I'll fix all of them in this file to match this changed one.